webrender_traits/src/api.rs

```diff
@@ -67,6 +69,15 @@ impl RenderApi {
         key
     }
 
+    ///
+    pub fn get_glyph_dimensions(&self, font_key: FontKey, size: Au, blur_radius: Au, index: u32)
```

Comment:
Thought about that too, but since GlyphKey is internal, we would either have to create a vector of tuples or a vector with an equivalent struct. Your thoughts?